cmd/plugin/plugin-scope-association.yaml

```diff
@@ -0,0 +1,7 @@
+plugins:
+- name: builder
+  target: global
+  scope: Standalone
+- name: test
+  target: global
+  scope: Standalone
```

Comment:
I see you tried to keep this case insensitive but eventually gave up.
I think it would be a nice improvement since this file is provided by users.  We know they will make the mistake of using `standalone` instead of `Standalone`.

Hmm, this made we wonder if having to specify `Standalone` or `Context` may be difficult for users unless they have an example.  Just an idea, could we change the field `scope: Standalone` to `isContextScoped: true`?  Would that be easier and less error-prone to users?

Reply:
Great suggestion. I have updated it to use the `isContextScoped` field.